reducers/shoppingReducer.js

```diff
@@ -0,0 +1,102 @@
+import {
+  FETCH_BANNERS,
+  FETCH_CATEGORIES,
+  FETCH_PRODUCTS,
+  SET_CATEGORY,
+  SET_CART_OPEN,
+  ADD_TO_SHIPPING,
+  DELETE_FROM_SHIPPING,
+  RESET_STORE,
+  SIGN_IN,
+  SIGN_OUT,
+} from "../actions/types";
+import _ from "lodash";
```

Comment:
Do not import lodash using _. instead call specific function from lodash.

Reply:
i have removed lodash, not using anymore